test/integration/simple_operations_test.lua

```diff
@@ -490,6 +490,193 @@ pgroup.test_intermediate_nullable_fields_update = function(g)
+        crud_replace_many = {
+            operation_name = 'crud.replace_many',
+            input = {'countries', {{3, box.NULL, 'vatican', 825}},
+                {fetch_latest_metadata = true}},
+            need_pre_insert_data = true,
+        },
+        crud_replace_object_many = {
+            operation_name = 'crud.replace_object_many',
+            input = {'countries',
+                {{id=3, bucket_id = box.NULL, name = 'vatican', population = 825}},
+                {fetch_latest_metadata = true}},
+            need_pre_insert_data = true,
+        },
+        crud_get = {
+            operation_name = 'crud.get',
+            input = {'countries', 3, {fetch_latest_metadata = true}},
+            need_pre_insert_data = true,
+        },
+        crud_delete = {
+            operation_name = 'crud.delete',
+            input = {'countries', 3, {fetch_latest_metadata = true}},
+            need_pre_insert_data = false,
+        },
+        crud_upsert = {
+            operation_name = 'crud.upsert',
+            input = {'countries',
+                {3, box.NULL, 'vatican', 825}, {{'+', 'population', 1}},
+                {fetch_latest_metadata = true}},
+            need_pre_insert_data = false,
+        },
+        crud_upsert_object = {
+            operation_name = 'crud.upsert_object',
+            input = {'countries',
+                {id=3, bucket_id = box.NULL, name = 'vatican', population = 825},
+                {{'+', 'population', 1}},
+                {fetch_latest_metadata = true}},
+            need_pre_insert_data = false,
+        },
+        crud_upsert_many = {
+            operation_name = 'crud.upsert_many',
+            input = {'countries',
+            {
+                {{3, box.NULL, 'vatican', 825},
+                {{'+', 'population', 1}}}
+            },
+            {fetch_latest_metadata = true}},
+            need_pre_insert_data = false,
+        },
+        crud_upsert_object_many = {
+            operation_name = 'crud.upsert_object_many',
+            input = {'countries',
+            {
+                {
+                    {id=3, bucket_id = box.NULL, name = 'vatican', population = 825},
+                    {{'+', 'population', 1}}
+                }
+            },
+            {fetch_latest_metadata = true}},
+            need_pre_insert_data = false,
+        },
+        crud_max = {
+            operation_name = 'crud.max',
+            input = {'countries', 'id', {fetch_latest_metadata = true}},
+            need_pre_insert_data = true,
+        },
+        crud_min = {
+            operation_name = 'crud.min',
+            input = {'countries', 'id', {fetch_latest_metadata = true}},
+            need_pre_insert_data = true,
+        },
+        crud_select = {
+            operation_name = 'crud.select',
+            input = {'countries', {}, {fetch_latest_metadata = true}},
+            need_pre_insert_data = true,
+        },
+    }
+
+    for case_name, case in pairs(cases) do
```

Comment:
Can you organize it to have a separate case for each operation?

For example, like here
https://github.com/tarantool/crud/blob/781ef16fc3ab725cf4c0720b9f67def80b32f785/test/integration/stats_test.lua#L591-L598